src/amo/components/Category/index.js

```diff
@@ -67,11 +68,17 @@ export function mapStateToProps(state, ownProps) {
     { ...state.search.filters, page: parsePage(state.search.page) },
     { ...filters, page: queryParams.page },
   );
+
+  const hasFetched = state.categories.hasFetched;
```

Comment:
It needs to be a prop as it's based on data in *all* categories but the prop that the component gets is just one category.